mods/cnc/chrome/settings-display.yaml

```diff
@@ -163,14 +163,26 @@ Container@DISPLAY_PANEL:
 					Height: 20
 					Children:
 						Container@UI_FEEDBACK_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 10
+							Children:
+								Checkbox@UI_FEEDBACK_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Show UI Feedback Notifications
+						Container@TRANSIENTS_CHECKBOX_CONTAINER:
 							X: PARENT_RIGHT / 2 + 10
 							Width: PARENT_RIGHT / 2 - 20
 							Children:
-								Checkbox@UI_FEEDBACK_CHECKBOX:
+								Checkbox@TRANSIENTS_CHECKBOX:
 									Width: PARENT_RIGHT
 									Height: 20
 									Font: Regular
-									Text: UI Feedback in Transients Panel
+									Text: Show Transient Notifications
+									TooltipText: Display text notifications for game events
```

Comment:
IMO it feels weird and is very poorly discoverable that this but literally nothing else in the display settings has a tooltip. What's wrong with calling the checkbox "Show Game Event Notifications"?